ui/src/components/features/build-panel/components/edit-assistant.tsx

```diff
@@ -20,13 +20,30 @@ const RetrievalType = "retrieval";
 
 export function EditAssistant() {
   const { assistantId } = useSlugRoutes();
-
   const { data: selectedAssistant, isLoading: isLoadingAssistant } =
     useAssistant(assistantId as string, {
       enabled: !!assistantId,
     });
 
-  const updateAssistant = useUpdateAssistant(selectedAssistant?.id as string);
+  return isLoadingAssistant ? (
```

Comment:
I am guessing this has something to do with the behavior I was seeing in System prompt... but yeah, I am utterly confused.